src/include/security/helps.rb

```diff
@@ -352,8 +331,37 @@ def initialize_security_helps(include_target)
         "EXTRA_SERVICES"                            => _(
           "<P>Every running service is a potential target of a security attack. Therefore it is recommended to turn off all services which are not used by the system.</P>"
         )
-      } 
+      }
+
+    end
+
+    def boot_dialog_help
+      help = _(
+        "<p><b><big>Boot Security</big></b></p>\n<p>In this dialog, change various boot settings related to security.</p>"
+      )
 
+      # Boot dialog help 2/4, not shown for s390 architecture
+      help << _(
+        "<p><b>Interpretation of Ctrl + Alt + Del</b>:\n" +
+          "Configure what the system should do in response to\n" +
+          "someone at the console pressing the CTRL + ALT + DEL key\n" +
+          "combination. Usually the system reboots. Sometimes it is desirable\n" +
+          "to ignore this event, for example, when the system serves as both\n" +
+          "workstation and server.</p>"
+      ) if !Arch.s390
```

Comment:
Ctrl + Alt + Del is possible on S390, but does not do a reboot - does shutdown instead.